src/layouts/Header/components/HelpDropdown/HelpDropdown.tsx

```diff
@@ -44,23 +50,7 @@ function HelpDropdown() {
     },
     {
       props: {
-        onClick: async () => {
-          const sentryFeedback = feedbackIntegration({
-            showBranding: false,
-            colorScheme: 'light',
-            formTitle: 'Give Feedback',
-            buttonLabel: 'Give Feedback',
-            submitButtonLabel: 'Send Feedback',
-            nameLabel: 'Username',
-            isEmailRequired: true,
-            autoInject: false,
-            id: 'help-dropdown-widget',
-            onFormClose: removeSentryForm,
-          })
-          const form = await sentryFeedback.createForm()
-          form.appendToDom()
-          form.open()
-        },
+        onClick: form?.open,
```

Comment:
Could also use the `isSuccess` here as well, and if not just an empty function?

Reply:
good call